src/state/mod.rs

```diff
@@ -399,25 +399,14 @@ impl IndexerState {
                 indexer_store.add_block(&precomputed_block)?;
 
                 if let Some(height) = precomputed_block.blockchain_length {
-                    let tmstmp = precomputed_block
-                        .protocol_state
-                        .body
-                        .clone()
-                        .inner()
-                        .inner()
-                        .blockchain_state
-                        .inner()
-                        .inner()
-                        .timestamp
-                        .inner()
-                        .inner();
-
-                    for cmd in precomputed_block.cmds() {
+                    let tmstmp = precomputed_block.timestamp();
+
+                    for cmd in precomputed_block.commands() {
                         indexer_store.put_tx(height, tmstmp, cmd)?;
                     }
                 }
 
-                // TODO store ledger at specified cadence, e.g. at epoch boundaries
+                // TODO: store ledger at specified cadence, e.g. at epoch boundaries
```

Comment:
litchrally me

Reply:
Just in case ya'll forgot! Jk this is just to make vs code highlight it